### PR TITLE
Add v0.4.0 of mattermost-plugin-alertmanager to the Marketplace

### PR DESCRIPTION
**Pull request by hanzei**
#### Ticket Link
Fixes https://github.com/mattermost/mattermost-marketplace/issues/316

### CODE REVIEW

**Comment by DHaussermann** [comment]
@hanzei this seem to work when I deploy to my local instance of the marketplace. 

But, 2/5 should this not be stamped as v`0.4.1` as 2 relatively significant issues have been solved from the v`0.4.0`?
Or are we complicating things if we allow the version number to go out of sync between the 2 repos.

**Comment by DHaussermann** [comment]
Sorry @asatkinson forgot to cc you in the comment above ☝️ in case you have thoughts.

**Comment by hanzei** [comment]
> Or are we complicating things if we allow the version number to go out of sync between the 2 repos

Yes, that is my thinking.

File: plugins.json [hunk 1]
```diff
@@ -1,4 +1,67 @@
...
+    "hosting": "",
+    "author_type": "community",
+    "release_stage": "beta",
+    "enterprise": false,
+    "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAmPheDgACgkQ0bVLR6XO/sQXww/9EAiDdkz3nUnfujEIigh2aIxRairSsoeyGAMjjrkLP0M5kAW/7iSbzHb4Fzcjg9kAUvrcEPL6Q5Xv5ZT59+fWF3Bq4Z9Zeb8LnsmQpXUxV93y0ZA56gfPHmviSdGu+l4vLKSk6wM1Go9ho1Rewku9wxXzkjNKu8r5Jlt+jgHQiRqEl7W2tqLfiLOstjg4ygksxrn41J8YvDYH+jMW2tbQjyPnjy+eWgDfG/iLv37J50s2hTbmdiDMhuUxjGx3SUeeF26emvyB6DmdsO8DJKbtH9xHLZvkLHWm5F7Sd/Irf7fE0Qj7woNiInNks4uDXUZaANVa64f+nJTc5ao98WGtyxy8/r/fMUfZCtVhV0/QcnPGw7mKA4Vk4/L/q675BHLLoMxNIsGULwV4K/mxGBAqKzWwMjeRpQPGwy+vLBteVECR8oyENtq1t2nS3uriV049hoBjSdwCItaPvIqFaKkUaVeLjUevdFUyEU2usXcuhUK0gSkYL9wjElDu27pVgOqz+lVJIrHmud3hIQreuMOBotKPjjqERV+auQGvMx/yEw7l4NBxRfZo9P5UW4nIeHGa+SIeYDGdKxJ2VuOBllnwbKfe/5mqqx2eG4/QjpNf/HC7yztUHQJAxCBphk5aZnyDo3VzxPeq/TapmTc3AcpCPCeR5Uo7NCTzWQY3pPE/8z4=",
+    "repo_name": "mattermost-plugin-alertmanager",
+    "manifest": {
+      "id": "alertmanager",
+      "name": "AlertManager",
+      "description": "Alermanager plugin for Mattermost, you can receive alerts and interact with alertmanager.",
```
Comment (mickmister) [inline plugins.json:16]:
```suggestion
      "description": "AlertManager plugin for Mattermost, you can receive alerts and interact with AlertManager.",
```

Comment (mickmister) [reply plugins.json:16]: I have a feeling some people won't know this is specifically related to Prometheus

File: plugins.json [hunk 2]
```diff
@@ -1,4 +1,67 @@
...
+    "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-alertmanager-v0.4.0.tar.gz",
+    "release_notes_url": "https://github.com/cpanato/mattermost-plugin-alertmanager/releases/tag/v0.4.0",
+    "hosting": "",
+    "author_type": "community",
+    "release_stage": "beta",
```
Comment (mickmister) [inline plugins.json:9]: @asatkinson Note that this is going into the marketplace as beta

File: plugins.json [hunk 3]
```diff
@@ -1,4 +1,67 @@
...
+    "manifest": {
+      "id": "alertmanager",
+      "name": "AlertManager",
+      "description": "Alermanager plugin for Mattermost, you can receive alerts and interact with alertmanager.",
+      "homepage_url": "https://github.com/cpanato/mattermost-plugin-alertmanager",
+      "support_url": "https://github.com/cpanato/mattermost-plugin-alertmanager/issues",
+      "release_notes_url": "https://github.com/cpanato/mattermost-plugin-alertmanager/releases/tag/v0.4.0",
+      "icon_path": "assets/alertmanager-logo.svg",
+      "version": "0.4.0",
+      "min_server_version": "7.5.0",
```
Comment (mickmister) [inline plugins.json:22]: Is this version actually required?

**Review by mickmister — APPROVED** [review]
Just a few comments, nothing blocking

File: plugins.json (hunk 3 above)
Comment (hanzei) [reply plugins.json:22]: Yes, due to the `EnsureBot` bug.

**Review by DHaussermann — APPROVED** [review]
Tested and passed
- Tested fresh install path
- Tested upgrade path (if web marketplace release was installed)
- Link correctly points to community repo
- App bar and marketplace icon has reasonable contrast in all themes 
- Version is showing as `0.4.0` as expected 

LGTM!

**Pull request merged**